aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -314,6 +317,9 @@ def parse_xform_to_avro_schema(
         if current_choices:
             current_field['@aether_lookup'] = current_choices
 
+        if current_extended_property:
```

Comment:
Also you don't need to check if the value exists, the `__clean_dict` method will remove the entry if the value is `None`. (This is valid also for the `if current_choices` line above)